src/test/resources/sampleheader1.txt

```diff
@@ -0,0 +1,11 @@
+col000000	col000001	col000002	col000003	col000004	col000005
+row000000	0	0	0	0	0	0
```

Comment:
Small tests with different values would be good, I think.  You can do them by hand, or generate random matrices with Gen that you read back and verify are identical.  You should be able to export matrices in this form by doing VariantDataset.make_table().export().